modules/ob-defining-the-service-account.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-runs.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-the-service-account_{context}"]
+= Define the service account
+
+You can define a service account in your `BuildRun` resource. The service account hosts all secrets referenced in your `Build` resource, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: shipwright.io/v1beta1
+kind: BuildRun
+metadata:
+  name: buildpack-nodejs-buildrun-namespaced
+spec:
+  build:
+    name: buildpack-nodejs-build-namespaced
+  serviceAccount: pipeline
+----
+
+You can also set the value of the `spec.serviceAccount` field to `".generate"` to generate the service account during runtime. The name of the generated service account corresponds with the name of the `BuildRun` resource.
```

Comment:
Would a user be setting it to `.generate` or `generate`?

Reply:
to `.generate`